src/Assetic/Filter/CleanCssFilter.php

```diff
@@ -237,96 +259,96 @@ public function filterLoad(AssetInterface $asset)
     /**
      * Run the asset through CleanCss
      *
-     * @see Assetic\Filter\FilterInterface::filterDump()
+     * @see FilterInterface::filterDump()
      */
     public function filterDump(AssetInterface $asset)
     {
-        $pb = $this->createProcessBuilder($this->nodeBin
+        $commandline = $this->nodeBin
             ? array($this->nodeBin, $this->cleanCssBin)
-            : array($this->cleanCssBin));
+            : array($this->cleanCssBin);
 
         if ($this->keepLineBreaks) {
-            $pb->add('--keep-line-breaks');
+            array_push($commandline, '--keep-line-breaks');
         }
 
         if ($this->compatibility) {
-            $pb->add('--compatibility ' .$this->compatibility);
+            array_push($commandline, '--compatibility ' . $this->compatibility);
```

Comment:
adding the option and its value separated by a space as a single element is likely to be broken after escaping (but I think the old code was buggy already)